fs/copy.py

```diff
@@ -74,38 +72,61 @@ def copy_fs_if_newer(
             a single-threaded copy.
 
     """
-    return copy_dir_if_newer(
-        src_fs, "/", dst_fs, "/", walker=walker, on_copy=on_copy, workers=workers
-    )
+    return copy_fs_if(src_fs, dst_fs, "newer", walker, on_copy, workers)
 
 
-def _source_is_newer(src_fs, src_path, dst_fs, dst_path):
-    # type: (FS, Text, FS, Text) -> bool
-    """Determine if source file is newer than destination file.
+def copy_fs_if(
+    src_fs,  # type: Union[FS, Text]
+    dst_fs,  # type: Union[FS, Text]
+    condition="always",  # type: Text
+    walker=None,  # type: Optional[Walker]
+    on_copy=None,  # type: Optional[_OnCopy]
+    workers=0,  # type: int
+):
+    # type: (...) -> None
+    """Copy the contents of one filesystem to another, depending on a condition.
 
-    Arguments:
-        src_fs (FS): Source filesystem (instance or URL).
-        src_path (str): Path to a file on the source filesystem.
-        dst_fs (FS): Destination filesystem (instance or URL).
-        dst_path (str): Path to a file on the destination filesystem.
+    Depending on the value of ``strategy``, certain conditions must be fulfilled
+    for a file to be copied to ``dst_fs``.
 
-    Returns:
-        bool: `True` if the source file is newer than the destination
-        file or file modification time cannot be determined, `False`
-        otherwise.
+    If ``condition`` has the value ``"newer"``, the last modification time
+    of the source file must be newer than that of the destination file.
+    If either file has no modification time, the copy is performed always.
+
+    If ``condition`` has the value ``"older"``, the last modification time
+    of the source file must be older than that of the destination file.
+    If either file has no modification time, the copy is performed always.
+
+    If ``condition`` has the value ``"exists"``, the source file is only
+    copied if a file of the same path already exists in ``dst_fs``.
+
+    If ``condition`` has the value ``"not_exists"``, the source file is only
+    copied if no file of the same path already exists in ``dst_fs``.
```

Comment:
Could you perhaps see if this looks better as a reStructuredText definition list? I'm thinking it may render a bit more nicely in the HTML documentation. This is done in `fs.glob` for instance (https://docs.pyfilesystem.org/en/latest/globbing.html):
```rst
Here's a summary of glob patterns:

``*``
    Matches all files in the current directory.
``*.py``
    Matches all .py file in the current directory.
``*.py?``
    Matches all .py files and .pyi, .pyc etc in the currenct directory.
```
(i'm noticing a typo there btw ahah)

Reply:
Done!